dashboard/app/models/script.rb

```diff
@@ -333,7 +333,7 @@ def beta?
   end
 
   def self.beta?(name)
-    name == 'edit-code' || name == 'gradeKinder' || name == 'grade1' || name == 'grade2' || name == 'grade3' || name == 'grade4' || name == 'grade5'
+    name == 'edit-code' || name == 'gradeKinder' || name == 'grade1' || name == 'grade2' || name == 'grade3' || name == 'grade4' || name == 'grade5' || name.start_with?('csd')
```

Comment:
It does look like being beta only affects how some things are displayed (i.e. we add Beta in the header, and on the tile when listed on the main page). I thought it might also be affecting things like whether the course showed up in the dropdown, but it doesnt seem like it does. As such, it might not be a big deal to accidentally mark a test script as beta.